src/DssSpell.sol

```diff
@@ -18,34 +18,36 @@ pragma solidity 0.8.16;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
-import { MCD, DssInstance } from "dss-test/MCD.sol";
-import { DssLitePsmInstance } from "./dependencies/dss-lite-psm/DssLitePsmInstance.sol";
-import { DssLitePsmMigrationPhase1, DssLitePsmMigrationConfigPhase1 } from "./dependencies/dss-lite-psm/phase-1/DssLitePsmMigrationPhase1.sol";
+import { GemAbstract } from "dss-interfaces/ERC/GemAbstract.sol";
 
-interface ProxyLike {
-    function exec(address target, bytes calldata args) external payable returns (bytes memory out);
+interface DssCronSequencerLike {
+    function addJob(address job) external;
+    function removeJob(address job) external;
 }
 
-interface PauseLike {
-    function setDelay(uint256 delay_) external;
+interface RwaLiquidationLike {
```

Comment:
Interface for the contract is available in [dss-interfaces](https://github.com/makerdao/dss-interfaces/blob/9bfd7afadd1f8c217ef05850b2555691786286cb/src/dss/mip21/RwaLiquidationOracleAbstract.sol#L6)

Reply:
As we only need one function here, I think it makes sense to use the `static interface` here 

https://github.com/makerdao/pe-checklists/blob/e2376d43fe55e606669c48b44c41b0b8e7de70cc/spell/spell-crafter-mainnet-workflow.md?plain=1#L92

Reply:
This actually clashes with an item on the reviewer checklist: https://github.com/makerdao/pe-checklists/blob/e2376d43fe55e606669c48b44c41b0b8e7de70cc/spell/spell-reviewer-mainnet-checklist.md?plain=1#L46C21-L46C34. 

I don't think we should deciding based on number of functions used, the compiler does not include the unused interface functions in bytecode, the spell looks cleaner by importing from dss-interfaces, and we're reusing trusted/lindy code.

I'll resolve this comment though based on the ambiguity and document this in the CL, and create an issue on the checklist repo to resolve this.

Reply:
https://github.com/makerdao/pe-checklists/issues/37